.gitignore

```diff
@@ -180,7 +180,6 @@ dmypy.json
 ### Python.VirtualEnv Stack ###
 # Virtualenv
 # http://iamzed.com/2009/05/07/a-primer-on-virtualenv/
-[Bb]in
```

Comment:
why?

Reply:
build-tools/bin is a legitimate path. And when I ran a git status, this wasn't filtering anything anyways.